frontend/apps/web/app/(mgmt)/[account]/settings/billing/page.tsx

```diff
@@ -1,4 +1,5 @@
 'use client';
```

Comment:
I originally touched this page because i was going to add account status stuff to it. Ended up only adding it to the site header.

However, it's now more organized, re-uses some components and is more type safe, as well as slightly more responsive.
There are still issues on smaller screens due to the left nav that need fixed (later).